### ISSUE TITLE
Option to scrape multiple numbers?

### ISSUE THREAD

**Issue by geekyadam1** [Sunil0612/Web-Scrap-of-JustDial-Website#1]
Sunil, the code works great. I was just wondering, let's say if we wanted to scrape more than one numbers, a lot of these listings have three to five numbers, could you please help in scraping multiple numbers?

**Comment by Sunil0612** [comment]
Sorry to say, But I am unable to get your question. Can you please
elaborate with an example?

**Comment by geekyadam1** [comment]
How to get 2-3 contact numbers from the listing rather than just one? The code only scrapes one mobile number from the listing, I would love if I could get all the contact information present on the listing(3-5 phone numbers generally).

**Comment by Sunil0612** [comment]
It's simple just check if there is another number or not. Let say there are
x no. of ph. numbers. So, loop it for x times and scrap x no. of phone
numbers.
May I know what's your name?

**Comment by geekyadam1** [comment]
Thank you, Sunil! My name is Adam.

I'm new to Python and still learning the basics, can you please share the exact code to replace whenever you're free?

**Comment by Sunil0612** [comment]
Okay Amit Yadav, You created this account just to message me, right?

You can contact me with your account as well. Anyway, Let's come to the point I will solve your issue soon.

**Comment by geekyadam1** [comment]
Looks like you're confusing me for someone else, I was trying to scrape a list of plushy hotels for NY, https://www.justdial.com/us/NY-New-York/5-Star-Hotels/page-5

Most of them just have one number and some don't even have a number so I'm not trying to maximise the conversions by trying to get the best results. Your help with some changes in the code would be really great to get the job done!

**Comment by Sunil0612** [comment]
But, you created your account on 1st April only. So, I thought you are
Amit, a guy who fork this repository.
Sure, I will rectify the code.

**Comment by Sunil0612** [comment]
As you mentioned that you want to scrap details of hotels in the US. I checked the link you provided. It's very simple in your case because the phone number is in text form.

**Comment by geekyadam1** [comment]
Yes, I noticed that too. I was playing with the code, some countries have
direct text while some have the characters which need to convert it, but to
make it globally friendly even in case of character conversion(backup
option), what kind of loop command do you think would work in such a case?

**Comment by Sunil0612** [comment]
I guess any loop can be used like for, while, do-while.

**Comment by geekyadam1** [comment]
Thanks, Sunil. I'll check out stackoverflow and some other repositories to
see how to input the while command in the code. :)

**Comment by Sunil0612** [comment]
In my case (India), there is only one phone number per hotel/chemist.
